server/models/Users.js

```diff
@@ -24,6 +25,53 @@ class Users {
       },
     }));
   }
+
+  static userLogin(req, res) {
```

Comment:
is there a reason why this implementation is in the UserModel instead of the User controller? Do you know what a Model is ? and a controller?